config/core-site.xml

```diff
@@ -2,6 +2,11 @@
 <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
 <configuration>
 
+  <property>
```

Comment:
I would comment this out and add it next to the WASB configuration and/or the ADLS configuration.